src/containers/Action/Worldmap/Map.js

```diff
@@ -0,0 +1,68 @@
+import {useCallback, useEffect, useState} from 'react';
+import Marks from './Marks';
+import worldmap from 'assets/images/world-map2.webp';
+
+const Map = ({width, color, backgroundColor}) => {
+    const [name, setName] = useState('');
+    const [data, setData] = useState('');
+    const [position, setPosition] = useState({x: 0, y: 0});
+
+    useEffect(() => {
+        const setPositionFromEvent = (e) =>
+            setPosition({x: e.clientX, y: e.clientY});
+        window.addEventListener('mousemove', setPositionFromEvent);
+
+        return () => {
+            window.removeEventListener('mousemove', setPositionFromEvent);
+        };
+    }, []);
+
+    useEffect(() => {
+        fetch(
+            'https://raw.githubusercontent.com/saugatdhimal/worldmap/main/world-map.json'
+        )
+            .then((data) => data.json())
+            .then((data) => setData(data));
+    }, []);
+
+    const countryName = useCallback((value) => {
+        setName(value);
+    }, []);
+
+    if (!data) {
+        return <img src={worldmap} alt='worldmap' style={{width: '100%'}} />;
```

Comment:
can't we show just empty d3 map (without marker) instead of image

Reply:
Is Loading..... ok?